config/tsconfig.base.json

```diff
@@ -2,6 +2,7 @@
   "compileOnSave": false,
   "compilerOptions": {
     "declaration": true,
+    "importHelpers": true,
```

Comment:
Because you are doing this in the base tsconfig, any helper fxns, across **all** packages, that match this flag will be required via `tslib`. As such, we need to add the `tslib` dependency to all packages using this base config not just the 3 using `async`/`await`

Reply:
You're right, done in 12c2b549d8696c592c79847d89e1aecad84f5fe2. I've omitted `*-types` packages as stated in the commit message.

Reply:
Perfect, thanks!